notebooks/official/explainable_ai/xai_text_classification_feature_attributions.ipynb

```diff
@@ -0,0 +1,844 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.                "encoding": aiplatform.explain.ExplanationMetadata.InputMetadata.Encoding(
</pre><p>What does Encoding(1) mean?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1801/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>

Reply:
<p>Identity.  The "Learn more about configuring feature-based explanations" link provides more details of the configuration </p>